package.json

```diff
@@ -9,6 +9,7 @@
     "plugins": [
       "prettier-plugin-organize-imports"
     ],
+    "organizeImportsSkipDestructiveCodeActions": true,
```

Comment:
Should we remove this? Don't think it's meant for this PR.

Reply:
Fixed thank you!